report_form_saver.py

```diff
@@ -193,33 +195,79 @@ def validate_email(self, email):
             flash(f"Email ({email}) is not valid. Please enter a valid email address.")
             return False
 
+    @staticmethod
+    def make_tz_aware(time):
+        """
+        Used for loading reports. To convert timezones we need to have a
+        timezone aware datetime for pendulum to convert from. Appends the
+        airflow default timezone to the time datetime object
+
+        :param time: time to make timezone-aware
+        :type time: datetime
+        """
+        default_tz = pendulum.timezone(conf.get("core", "default_timezone"))
+        time_of_day_to_local = pendulum.datetime(
+            1970,
+            1,
+            1,
+            time.hour,
+            time.minute,
+            tzinfo=default_tz
+        )
+        return time_of_day_to_local
+
+    def convert_to_default_timezone(self, time):
+        """
+        Uses the schedule timezone provided from the form
+        to convert the time provided into the default airflow timezone
+        in the backend. All times are stored as UTC in the variable.
+
+        :param time: Time to convert to the default timezone
+        :type time: datetime.time
+        """
+        default_tz = pendulum.timezone(conf.get("core", "default_timezone"))
+
+        time_of_day_to_local = pendulum.datetime(
+            1970,
+            1,
+            1,
+            time.hour,
+            time.minute,
+            tzinfo=self.report_dict["schedule_timezone"]
+        )
+        time_of_day_to_utc = time_of_day_to_local.in_timezone(default_tz)
```

Comment:
on the client side you're assuming that backend always stores time in UTC
`let dateTimeObj = moment.utc(`${defaultDate} ${time}`);`
but in fact it can be different than UTC if changes in the config

Reply:
ahhhh looking into this

Reply:
i think we need to only store as utc

Reply:
i agree with that

Reply:
addressed [here](https://github.com/Raybeam/rb_status_plugin/pull/140/commits/23a6689d75d80c2ed269c76f662b2cd5203f86a8)